client/state/sites/test/selectors.js

```diff
@@ -1047,4 +1073,405 @@ describe( 'selectors', () => {
 			expect( showcasePath ).to.eql( '/theme/journalistic/setup/testonesite2014.wordpress.com' );
 		} );
 	} );
+
+	describe( '#siteHasMinimumJetpackVersion()', () => {
+		it( 'it should return `null` for a non-existing site', () => {
+			const state = {
+				sites: {
+					items: {}
+				}
+			};
+			let siteId;
+
+			const hasMinimumVersion = siteHasMinimumJetpackVersion( state, siteId );
+			expect( hasMinimumVersion ).to.equal( null );
+		} );
+
+		it( 'it should return `null` for a non jetpack site', () => {
+			const siteId = 77203074;
+
+			const state = {
+				sites: {
+					items: {
+						77203074: {
+							ID: siteId,
+							jetpack: false,
+						}
+					}
+				}
+			};
+
+			const hasMinimumVersion = siteHasMinimumJetpackVersion( state, siteId );
+			expect( hasMinimumVersion ).to.equal( null );
+		} );
+
+		it( 'it should return `true` if jetpack version is greater that minimum version', () => {
+			const jetpackMinVersion = config( 'jetpack_min_version' );
+			const greaterVersion = changeVersion( jetpackMinVersion );
```

Comment:
I think it's easier to read with explicit test data. We can instead mock `jetpackMinVersion` to a fixed value instead of adding to the real one.